src/pages/contact_info.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { Card, Image, Text, Badge, Button, Group, Grid, CardSection } from "@mantine/core";
+
+interface ContactCardProps {
+	title: string;
+	operationTitle: string;
+	operationTime: string;
+	imgUrl: string;
+}
+
+const contactCards = [
+	{
+		imgUrl: "/assets/images/contact_info/Rectangle 1.png",
```

Comment:
We already have similar image stored in the logo directory, use this one instead `public/assets/images/logos/blackboard-logo.png` or this one `public/assets/images/logos/registrar.png` (maybe rename it to something else more universal like stamford.png)